src/geojsonm/geojsonm.mli

```diff
@@ -79,3 +79,20 @@ val fold_geometry :
 val fold_props :
   ('a -> Ezjsonm.value -> 'a) -> 'a -> Jsonm.src -> ('a, Err.t) result
 (** [fold_props f init src] *)
+
+(** {2 Iterators}
+
+    A standard usage of 'Iterators' is as follows:
+
+    val iter : ('a -> unit) -> 'a list -> unit
```

Comment:
Could we have a concrete example here instead of a type signature. A type isn't really a standard usage but something like:

```ocaml
List.iter print_int [1; 2; 3; 4]
```